aidants_connect_carto_api/utilities.py

```diff
@@ -143,6 +194,36 @@ def sanitize_opening_hours_with_hoh(opening_hours: str):
 # Other fields
 
 
+def process_public_target(value: str):
+    """
+    Tout public
+    Demandeurs d'emploi
+    Adhérents
+    Séniors
+    Assurés sociaux
+    jeunes
+    Jeunes entre 16 et 25 ans
+    Enseignants, formateurs jeunesses, membres associatifs
+    Allocataires CAF
+    Familles allocataires avec quotient familial (QF) inférieur à 800
+    """
+    public_target_list = []
+    if any(elem in value.lower() for elem in ["public"]):
```

Comment:
En première approche, ce code me donne des envies de DRY, mais en même temps, je ne suis pas sûr que la version factorisée serait _nécessairement_ plus lisible :)

Reply:
en effet c'est hacky au possible actuellement, mais c'est pour couvrir le maximum de cas dans un minimum de temps (80-20). de notre coté itérer sur les valeurs possibles en fonction de la donnée que l'on rencontre. et ensuite fournir ca aux hubs pour qu'ils corrigent / améliorent leur donnée :)